themes/default/content/docs/intro/console/project-and-stack-management.md

```diff
@@ -7,7 +7,9 @@ menu:
     weight: 4
 ---
 
-> While you can create projects within the individual organization created for your account, instructions in this document apply to members of an organization with sufficient [stack permissions]({{< relref "stack-permissions" >}}).
+{{% notes "info" %}}
+While you can create projects within an individual account, instructions in this document apply to members of an organization with sufficient [stack permissions]({{< relref "stack-permissions" >}}).
```

Comment:
Totally not your PR, but `relref`s should **always** be root-relative because of some odd quirks in the way hugo resolves its page-relative references. When two pages in a Hugo site share the same name (e.g., two "stack-reference.md" files, at different places in the content hierarchy), Hugo sometimes-but-not-always fails with "ambiguous refeference" errors that are generally vague and hard to track down. If you run across these in the future, please make them root-relative. 🙏 

```suggestion
While you can create projects within an individual account, instructions in this document apply to members of an organization with sufficient [stack permissions]({{< relref "/docs/intro/console/stack-permissions" >}}).
```